helmfile/upstream/kube-prometheus-stack/values.yaml

```diff
@@ -1123,8 +1439,8 @@ kubeEtcd:
   ##
   service:
     enabled: true
-    port: 2379
-    targetPort: 2379
+    port: 2381
+    targetPort: 2381
```

Comment:
Is this true for us?

Reply:
no, but we are overwriting it:
[sc](https://github.com/elastisys/compliantkubernetes-apps/blob/main/helmfile/values/kube-prometheus-stack-sc.yaml.gotmpl#L21:L24)
[wc](https://github.com/elastisys/compliantkubernetes-apps/blob/main/helmfile/values/kube-prometheus-stack-wc.yaml.gotmpl#L43:L46)

Reply:
Ah, missed that. Thanks